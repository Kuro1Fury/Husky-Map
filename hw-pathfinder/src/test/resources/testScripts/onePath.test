# Test two nodes with only one edge

CreateGraph A

AddNode A n1
AddNode A n2

AddEdge A n1 n2 1.0

FindPath A n1 n2