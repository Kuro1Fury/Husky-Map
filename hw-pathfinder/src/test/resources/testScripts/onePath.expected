# Test two nodes with only one edge

created graph A

added node n1 to A
added node n2 to A

added edge 1.000 from n1 to n2 in A

path from n1 to n2:
n1 to n2 with weight 1.000
total cost: 1.000